### ISSUE TITLE
Further exploration 4 (E-grade)

### ISSUE THREAD

**Issue by Atheer2104** [Atheer2104/dd2424-CNN-classificaiton-project#8]
It is not clear in which order Dropout and Batch Norm should be performed. In the tutorial 
it is BatchNorm then dropout. Check if changing the order to Batch Norm then Dropout 
has an effect on performance. Also check if the Dropout and Batch Norm are complementary 
ie having both Dropout and Batch Norm in the network is better or worse than having a 
network that just has one of these regularization techniques.
